examples/gno.land/p/demo/ownable/ownable.gno

```diff
@@ -37,8 +37,8 @@ func (o *Ownable) TransferOwnership(newOwner std.Address) error {
 	o.owner = newOwner
 	std.Emit(
 		OwnershipTransferEvent,
-		"from", string(prevOwner),
-		"to", string(newOwner),
+		"from", prevOwner.String(),
```

Comment:
Is not the same as before?

Reply:
yes, just thought its better to use the methods available on `std.Address`